includes/admin/class-wc-rest-stripe-account-keys-controller.php

```diff
@@ -399,8 +399,10 @@ public function configure_webhooks( WP_REST_Request $request ) {
 			$secret = $saved_secret;
 		}
```

Comment:
We should remove this code, and always use the secret from the settings.

This was necessary when the modal allowed manual keys to be entered, but currently, there is no reason to allow a manual secret key (aka shared key) here.